languages.yaml

```diff
@@ -1071,6 +1071,38 @@ Hcl:
       - "#5F43E9"
       - "#4040B2"
     chip: "#AACE60"
+Hlsl:
+  type: programming
+  ascii: |
+    {0}                  $$
+    {0}               $$$$$$$$
+    {0}          $$$$$$$$$$$$$$$$$$         $$$
+    {0} $$$$$$$$$$$$$$$$$$$$$$$$$$$$      $$$$$
+    {0}${1}$$$$${0}$$$$$$$$$$$$$$$$$$$$$$$$$    $${1}$
+    {0}$$     $$$$$$$$$$$$$$$$$$$$$$$$$ $$$${1}$
+    {0}$$     $$$$$$$$$$$$$$$$$$$$$$$$$$$${1}$$$
+    {0}$$$$   $$$$$$$$$$$$$$$$$$$$$$$$$$${1}$$
+    {0} $$$$$$$$$$$$$$$$$$$$$$$$$$$$$$${1}$$$
+    {0}   $$$$$$$$$$$$$$$$$$$$$$$$$${1}$$$$$
+    {0}     $$$$$$$$$$$$$$$$$$$${1}$$$$$$$
+    {0}      {1}$$$$$$$$$$$$${1}$$$$$$$$$$$
+    {0}         {1}$$$$$$$$$$$$$$$$$$
+    {2}       _      _  _        _____  _   
+    {2}      |:|    |:||:|      /:::::||:|
+    {2}      |:|____|:||:|     |:(___  |:|
+    {2}      |::::::::||:|      \::::\ |:|
+    {2}      |:|    |:||:|____  ____):||:|____
+    {2}      |:|    |:||::::::||:::::/ |::::::|
```

Comment:
IMO it looks a bit unusual with borders on the left, right, and top sides of the letters, but not the bottom. Have you considered adding a bottom border?
```
    {2}       _      _  _        _____  _   
    {2}      |:|    |:||:|      /:____||:|
    {2}      |:|____|:||:|     |:(___  |:|
    {2}      |::____::||:|      \___:\ |:|
    {2}      |:|    |:||:|____  ____):||:|____
    {2}      |_|    |_||______||_____/ |______|
```

This is just a basic idea. It sacrifices some of the "filling," but you might be able to get creative with some other characters to get the best of both worlds.

Reply:
Done